karma.conf.js

```diff
@@ -0,0 +1,114 @@
+'use strict'
+
+const fs = require('fs')
+
+module.exports = function (karma) {
+  const glob = 'tests/browser/*.js'
+  const files = [{pattern: glob}]
+
+  const customLaunchers = {
+    SL_Edge: {
+      base: 'SauceLabs',
+      platform: 'Windows 10',
+      browserName: 'microsoftedge'
+    },
+    SL_Firefox: {
+      base: 'SauceLabs',
+      browserName: 'Firefox'
+    },
+    SL_Chrome: {
+      base: 'SauceLabs',
+      browserName: 'Chrome',
+      platform: 'Windows 10'
+    },
+    SL_Safari: {
+      base: 'SauceLabs',
+      browserName: 'safari',
+      platform: 'OSX 10.11'
+    },
+    SL_IE11: {
+      base: 'SauceLabs',
+      browserName: 'internet explorer',
+      version: '11'
+    },
+    SL_IE10: {
+      base: 'SauceLabs',
+      browserName: 'internet explorer',
+      version: '10'
+    },
+    SL_IE9: {
+      base: 'SauceLabs',
+      browserName: 'internet explorer',
+      version: '9'
+    }
+  }
+
+  const sauceConfig = {
+    sauceLabs: {
+      testName: 'choo browser tests',
+      recordScreenshots: false,
+      connectOptions: {
+        port: 5757,
+        logfile: 'sauce_connect.log'
+      },
+      public: 'public'
+    },
+    captureTimeout: 120000,
+    customLaunchers: customLaunchers,
+    browsers: Object.keys(customLaunchers)
+  }
+
+  let config = {
+    frameworks: ['tap', 'browserify'],
+    preprocessors: {
+      [glob]: ['browserify']
+    },
+    reporters: ['dots'],
+    port: 9876,
+    colors: true,
+    logLevel: karma.LOG_INFO,
+    files: files,
+    browserify: {
+      debug: true,
+      transform: ['sheetify/transform', 'es2020']
```

Comment:
ah yeah, `es2020` should be under the `"global"` keyword here, as `transform` only applies to code declared in the repo :sparkles: